.github/blunderbuss.yml

```diff
@@ -0,0 +1,4 @@
+assign_issues:
+  - GoogleCloudPlatform/observability-devx
```

Comment:
How does it work? Will it assign issues with round robin logic to this group members?

Reply:
Correct, it should round robin to this group's members. Same as the repos with `googleapis/api-logging` team designated